### PR TITLE
Show payment related errors on team invitation

### LINKED ISSUE #1083: Show needed errors on team invitation

### PR DESCRIPTION
**Pull request by ArsalanSavand**
Fixes #1083 

# Screenshot

image

### CODE REVIEW

File: src/app/team/invite_modal/invite_modal_controller.js [hunk 1]
```diff
@@ -36,6 +36,14 @@ function TeamInviteModalController($scope, $rootScope, toaster, API, AuthService
         });
       },
       function(data) {
+        // Check if blog has reached maximum number of team members.
+        if (data.data.non_field_errors[0]) {
```
Comment (Alir3z4) [inline src/app/team/invite_modal/invite_modal_controller.js]: This is confusing.
The error that comes back from the server is:
```
{"non_field_errors":["You have reached the max number of allowed team members, please upgrade."]}
```

You don't need to rewrite and re-translate it.
Once we have localizations both backend and frontend will do be in sync.,

Just show the error that comes from the backend directly.

Comment (ArsalanSavand) [reply src/app/team/invite_modal/invite_modal_controller.js]: Oh yea my bad.

File: src/public/languages/en.json [hunk 2]
```diff
@@ -127,6 +127,7 @@
   "INVITED": "Invited",
   "REMOVE_FROM_TEAM": "Remove from team",
   "TEAM_MEMBERS": "Team members",
+  "REACHED_MAX_TEAM_MEMBER": "You have reached the max number of allowed team members, please upgrade.",
```
Comment (AmirSavand) [inline src/public/languages/en.json]: Change to "maximum".

File: src/app/team/invite_modal/invite_modal_controller.js (hunk 1 above)
Comment (ArsalanSavand) [reply src/app/team/invite_modal/invite_modal_controller.js]: Done.

**Pull request merged**